### ISSUE TITLE
TrackMe bug report - DHM tracker resets lag threshold values to default

### ISSUE THREAD

**Issue by sebwurl** [trackme-limited/trackme-report-issues#44]
I can't narrow it down exactly, but under some circumstances the customized lag thresholds for DHM entities are reset to the default value by the hybrid tracker. In the past I changed the default threshold values in the config tab and re-created the custom tracker. Not sure whether this is relevant for this behaviour.

I'm on version 2.0.12.

**Comment by guilhemmarchand** [comment]
Thanks @sebwurl 
Will try to reproduce - I haven't noticed this but will investigate

**Comment by guilhemmarchand** [comment]
This should not be happening anymore - nor being posible since the introduction of the persistent field custom command @sebwurl 
Let us know otherwise ;-)

**Comment by sebwurl** [comment]
I updated to v.2.0.19 and re-created trackers as recommended for fix #69, but this behavior still occurs. I am not able to change the latency value per host in DHM.

I recognized that the threshold is not resetted to my default set on the configuration page.
I changed the configuration page like this:

![image](https://user-images.githubusercontent.com/85566586/231736644-689ff2e7-b6f8-4958-8227-d7481f6b2e50.png)

But in the macros I see an historical default value I defined some time ago, which is then used as the actual threshold.
![image](https://user-images.githubusercontent.com/85566586/231737132-edc92784-ae8c-4726-abf0-e7f4539001ae.png)

That does not explain to me, why this value is finally used instead of the custom threshold of the entity, but it looks suspicious.
